.gitignore

```diff
@@ -2,6 +2,7 @@
 *.retry
 pontus.larsson_medsci.uu.se.key
 charon_credentials.yml
-statusdb_creds.yml
+statusdb_creds_prod.yml
```

Comment:
No. That's why I'm asking. Because then the file will be named differently when actually deployed on the cluster. And then the software reading this file will have to know the name of it, and that it will change (and to what). But if it is enough with having that environment variable set then I guess there are no worries.